test/scripts/e2e/test_kfservice_creating.py

```diff
@@ -0,0 +1,19 @@
+import time
+import os
+from kubeflow.testing import util
+
+def test_tensorflow_kfservice():
```

Comment:
IMO we should either use the golang client or the python client for these tests. I expect as our test suite grows, we will want some libraries to encapsulate some of the code. Having a well factored code base will help us grow closer to 100% coverage.

Reply:
@ellis-bigelow @yuzisun Yes, Yesterday I used python client to create kfservice, but found we cannot do that directly. Because we execute this in the `test-worker`, and we create new cluster and deploy cluster on the kfservice, after `gcloud container clusters get-credentials new_cluster ...`.  
Yes I totally agree with you, that's better to use python client. Two ways may works, will get a try:
1. Spcify the kube-conf of created cluster for `KFServingClient` (need to confirm if that works)
2. Create a work flow, and let kubectl to create the workflow to created cluster, the work flow can do e2e tests.

Reply:
Used python client.